### PR TITLE
fix(aquavm): temporary fix entire value in canon

### PR DESCRIPTION
**Pull request by mikevoronov**

### CODE REVIEW

File: crates/air-lib/interpreter-data/src/executed_state/impls.rs [hunk 1]
```diff
@@ -129,8 +129,8 @@ impl std::fmt::Display for ExecutedState {
             Ap(ap) => {
                 write!(f, "ap: _ -> {:?}", ap.res_generations)
             }
-            Canon(canon) => {
-                write!(f, "canon {:?}", canon.stream_elements_pos)
+            Canon(_) => {
+                write!(f, "canon [<object>]")
```
Comment (monoid) [inline crates/air-lib/interpreter-data/src/executed_state/impls.rs:133]: We loose information.

File: crates/air-lib/trace-handler/src/merger/canon_merger.rs [hunk 2]
```diff
@@ -58,75 +51,33 @@ pub(crate) fn try_merge_next_state_as_canon(data_keeper: &mut DataKeeper) -> Mer
...
     })
 }
 
 fn check_canon_results(
     prev_canon_result: &CanonResult,
     current_canon_result: &CanonResult,
-    data_keeper: &DataKeeper,
 ) -> Result<(), CanonResultError> {
-    if prev_canon_result.stream_elements_pos.len() != current_canon_result.stream_elements_pos.len() {
+    if prev_canon_result.canonicalized_element.len() != current_canon_result.canonicalized_element.len() {
```
Comment (monoid) [inline crates/air-lib/trace-handler/src/merger/canon_merger.rs:69]: We might compare content, if you care.

**Review by monoid — APPROVED** [review]

File: crates/air-lib/test-utils/src/executed_state.rs [hunk 3]
```diff
@@ -137,7 +137,12 @@ pub fn ap(generation: u32) -> ExecutedState {
     ExecutedState::Ap(ap_result)
 }
 
-pub fn canon(canonicalized_element: Vec<u8>) -> ExecutedState {
+pub fn canon(_canonicalized_element: Vec<u8>) -> ExecutedState {
```
Comment (monoid) [inline crates/air-lib/test-utils/src/executed_state.rs:140]: Variable is unused. Is it valid? Is the function used?

Comment (mikevoronov) [reply crates/air-lib/test-utils/src/executed_state.rs]: Thanks, forgot to delete.

**Pull request merged**